libmcount/plthook.c

```diff
@@ -732,16 +732,17 @@ static unsigned long __plthook_entry(unsigned long *ret_addr,
 	struct sym *sym;
 	struct mcount_thread_data *mtdp = NULL;
 	struct mcount_ret_stack *rstack;
-	struct uftrace_trigger tr = {
-		.flags = 0,
-	};
 	bool skip = false;
 	bool recursion = true;
 	enum filter_result filtered;
 	struct plthook_data *pd;
 	struct plthook_special_func *func;
 	unsigned long special_flag = 0;
 	unsigned long real_addr = 0;
+	struct uftrace_trigger tr;
+
+	mcount_memset4(&tr, 0, sizeof(tr));
+	tr.flags = 0;
```

Comment:
You don't need to reset flag anymore.. :)

Reply:
I left it to keep the original code, but removed and updated it anyway.